python/sdist/amici/petab_objective.py

```diff
@@ -153,6 +164,29 @@ def simulate_petab(
 
     # Compute total llh
     llh = sum(rdata['llh'] for rdata in rdatas)
+    # Compute total sllh
+    sllh = None
+    if solver.getSensitivityOrder() != amici.SensitivityOrder.none:
+        sllh = aggregate_sllh(
+            amici_model=amici_model,
+            rdatas=rdatas,
+            parameter_mapping=parameter_mapping,
+            petab_scale=scaled_parameters,
+            petab_problem=petab_problem,
+            edatas=edatas,
+        )
+        if not scaled_gradients:
```

Comment:
```suggestion
        if not scaled_gradients and sllh is not None:
```

may be None in case of integration errors. see notebook failures on RTD.